pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -707,6 +729,8 @@ message Image {
     repeated string repo_digests = 3;
     // The size of the image in bytes.
     optional uint64 size = 4;
+    // The uid that will run the command(s).
+    optional int64 uid = 5;
```

Comment:
Yeah, we have that image info handy as well.

I'm :+1: for kubelet doing that check.

My nit for this field is that `user` probably makes more sense since it can be either a name or uid, and the comment needs to clarify that dual-meaning as well.